OneSignalSDK/onesignal/build.gradle

```diff
@@ -2,11 +2,6 @@ ext {
     buildVersions = [
             minSdkVersion: 16
     ]
-
-    androidXVersion = '[1.0.0, 1.99.99]'
```

Comment:
Could we have these variables updated instead? Or if there. Any reason behind deleting the variables?

Reply:
We would have to add new ones for each AndroidX library. I removed them instead and we never override or reassign them and I don't see a reason in the future we would either. It is much easier to read and update versions this way if there isn't a reason to use variables for them.